### PR TITLE
Move subheading to be included in the h1

### PR DESCRIPTION
**Pull request by aliuk2012**
# Context

- https://dsdmoj.atlassian.net/browse/CAS-471

# Changes in this PR

Subheading in the context they are being used would be very important for screenreaders to inform users what page they are viewing.

Example, If this change was not done then screenreaders would only read out "Joe Bloggs Unallocated" instead of  "Referral summary Joe Bloggs "

ref: https://design-system.service.gov.uk/styles/headings/#headings-with-captions
If the caption should be considered part of the page heading, you can also nest the caption within the <h1>.

There should be no visional difference from what users previously saw.

# Release checklist

[Release process documentation](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/edit-v2/4247847062?draftShareId=a1c360ab-bd31-4db1-aae3-7cc002761de9).

## Pre-merge checklist

- [ ] Are any changes required to dependent services for this change to work?
  (eg. CAS API)
- [ ] Have they been released to production already?

## Post-merge checklist

- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to test
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to preprod
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to prod

### CODE REVIEW

File: server/views/temporary-accommodation/assessments/summary.njk [hunk 1]
```diff
@@ -33,9 +33,11 @@
         <div class="moj-identity-bar__container">
             <div class="moj-identity-bar__details">
         <span class="moj-identity-bar__title">
-          <span class="govuk-caption-xl">Referral summary</span>
-          <h1 class="govuk-heading-xl">{{ name }}
-              {{ assessmentStatusTag(assessment.status) | safe }}</h1>
+            <h1 class="govuk-heading-l">
+                <span class="govuk-caption-l">Referral summary</span>
```
Comment (froddd) [inline server/views/temporary-accommodation/assessments/summary.njk]: I don't think the change of size modifier from `-xl` to `-l` is desirable here? This does change the rendered size.

**Review by froddd — CHANGES_REQUESTED** [review]

File: server/views/temporary-accommodation/assessments/summary.njk (hunk 1 above)
Comment (aliuk2012) [reply server/views/temporary-accommodation/assessments/summary.njk]: Fixed now.

**Comment by froddd** [comment]
Just came across this extra caption which should also be moved into the following heading: https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/blob/801f1fa30e9f48ebf503ea4923dacf08fa3afaf0/server/views/applications/full.njk#L28

**Review by froddd — APPROVED** [review]
👍

**Pull request merged**